flux-kustomize-controller.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: flux-kustomize-controller
   version: 1.3.0
-  epoch: 2
+  epoch: 1
```

Comment:
When adding a test there's no need to bump the epoch since the package isn't rebuilt